tests/conftest.py

```diff
@@ -32,7 +31,8 @@
 from app.hasher import pbkdf2_sha256
 from tests.db.models import ModelForTests, ParentModelForTests  # noqa: F401
 from tests.types import ModelFactory
-from tests.utils import SQLAlchemyCapturer
+
+pytest_plugins = ["pytest_capsqlalchemy"]
```

Comment:
This can be avoided using setuptools entrypoints in the plugin. See https://docs.pytest.org/en/stable/how-to/writing_plugins.html#making-your-plugin-installable-by-others